docs.php

```
@@ -207,12 +178,72 @@ function extract_info(string $class_name, array $exclude = [], string $base_path
+ *   The comment.
+ *
+ * @return array<string, string>
+ *   Array of 'description' and 'description_full' keys.
+ */
+function parse_class_comment(string $trait_name, string $comment): array {
+  if (empty($comment)) {
+    throw new \Exception(sprintf('Class comment for %s is empty', $trait_name));
+  }
+
+  $comment = preg_replace('#^/\*\*|^\s*\*\/$#m', '', $comment);
+  $lines = array_values(
```

PHPStan (2.0.3)

Parameter #1 $array (non-empty-list<string>) of array_values is already a list, call has no effect.
Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your %configurationFile%.

(arrayValues.list)